src/main/java/book/service/BookApp.java

```diff
@@ -0,0 +1,216 @@
+package book.service;
+
+import org.store.book.Book;
+import org.store.book.BookStatus;
+import org.store.book.OwnerDTO;
+import org.store.book.User;
+
+import java.text.ParseException;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static validation.ValidationForBookAndUser.convertDate;
+public class BookApp {
+    public void bookAppDashboard(User user, Map<Integer,OwnerDTO> owners, Map<String,String> request, Map<String,List<Book>> bookStore){
+        Scanner scanner = new Scanner(System.in);
+        Map<Integer,OwnerDTO> bookShare = new HashMap<>();
+        boolean logout = false;
+
+
+        while (!logout) {
+            System.out.println("Please choose the below Options");
+            System.out.println("1:     To add the new BOOK                        2:     To get book Information by Name of the Book ");
+            System.out.println("3:     To Get Book from the store by author       4:      To get Book Information by key word          ");
+            System.out.println("5:     To Request a book                          6:       To Return book.                           ");
+            System.out.println("7:     To exit");
+
+            try {
```

Comment:
Extract private methods from the body of all if-else statements for maintainability